docs/wiki/feature/pylons.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: Pylons
+description: Adds an interface that allow players to configure aircraft pylons.
+group: feature
+category: interaction
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 11
+  patch: 0
+---
+
+## 1. Overview
+
+Adds an interface that allow players to configure aircraft pylons. Loadouts can be saved and loaded through interface.
+
+## 2. Usage
+
+### 2.1 Configure Pylons
+
+To configure pylons you need a `Toolkit` equipped.
```

Comment:
There also has to be a rearm vehicle nearby. Both of those requirements can also be disabled.

Reply:
☝️  And a require engineer requirement can be enabled.